identity-wallet/bindings/HistoryCredential.ts

```diff
@@ -0,0 +1,3 @@
+// This file was generated by [ts-rs](https://github.com/Aleph-Alpha/ts-rs). Do not edit this file manually.
+
+export interface HistoryCredential { title: string, sub_title: string, id: string, }
```

Comment:
`sub_title` is non-conventional naming for JS/TS. It always contains the `issuerName`, right? So you could also just use that. If you want to keep it more generic, you could also name it `description` (as in the generic `ListItemCard`).